seed/index.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const _ = require('underscore');
+
+// connect to mongodb
+mongoose.connect('mongodb://localhost/airbnb', {useNewUrlParser: true, useUnifiedTopology: true});
+
+const db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error: '));
+db.once('open', function() {
+  console.log('We are connected!');
+});
+
+// define more places to stay schema
+const morePlacesSchema = new mongoose.Schema({
+  morePlaces: [Number],
+  propertyId: Number
+});
+
+// compile schema into model
+const MorePlaces = mongoose.model('MorePlaces', morePlacesSchema);
+
+const seedMorePlaces = async function() {
+  await MorePlaces.deleteMany({});
+
+  const range = _.range(1, 101);
+  _.each(range, async (element, index, list) => {
+    const container = [];
+    while (container.length < 12) {
+      container.push(Math.floor(Math.random() * (101 - 1) + 1));
+    }
+
+    const newMorePlaces = await new MorePlaces({
+      morePlaces: container,
+      propertyId: element
+    });
```

Comment:
to confirm you're storing an array of property id's chosen randomly and eventually we'll use teammates routes to get all the information on the page?

Reply:
@GalexyN correct. per our [service plan](https://docs.google.com/document/d/12ntpB_h4dH7rCNg7iVg0H0LeBaZdO3SzKLjoiu47QVg/edit#heading=h.jfv71j6gs9c1), i will have to request data from other services. for my service, i will simply store an array of property ids.